test/witan/models/dem/ccm/inputs_test.clj

```diff
@@ -7,4 +7,4 @@
   (let [r (sut/in-hist-deaths-by-age-and-sex-1-0-0
            nil {:src "./datasets/test_datasets/model_inputs/mort/bristol_hist_deaths_mye.csv"
                 :fn (partial wt/local-download (:witan/metadata (meta #'sut/in-hist-deaths-by-age-and-sex-1-0-0)))})]
-    (is (contains? r :historic-deaths))))
+    (is (contains? r :historic-deaths) "If this test fails, you probably need to run lein split-data first.")))
```

Comment:
Apparently you now need to do this for the tests to run. It has been added to Circle CI but there was no documentation about it so my local tests failed until @mantree figured it out and suggested adding this message.